gnosis/safe-react
# Fix: xDai -> Gnosis Chain

## What it solves
Resolves #3200

## How this PR fixes it
* Renames `XDAI` to `GNOSIS_CHAIN` in the `CHAIN_ID` enum
* Adds a redirect from `/app/xdai:0x...` to `/app/gno:0x...`
* Adjusts `setChainFromUrl` so that any redirects happen only from the router
* Updates `/app/xdai` to redirect to `/app/gno`
* Adds all `shortNames` as 'root' routes that set the `chainId` on app load

## How to test it
* Switch the CGW toggle to prod
* Open some Gnosis Chain Safe
* Try replacing `gno` with `xdai` in the URL – it should redirect back to `gno`
